ibm/mas_devops/roles/suite_db2_setup_for_manage/templates/setupdb.sh.j2

```diff
@@ -68,9 +68,9 @@ db2set DB2_USE_ALTERNATE_PAGE_CLEANING=ON
 # so for now we will ignore failures in them
 set +e
 db2 CREATE BUFFERPOOL MAXBUFPOOL IMMEDIATE SIZE 4096 AUTOMATIC PAGESIZE 32 K
-db2 CREATE REGULAR TABLESPACE MAXDATA PAGESIZE 32 K MANAGED BY AUTOMATIC STORAGE INITIALSIZE 5000 M BUFFERPOOL MAXBUFPOOL
+db2 CREATE REGULAR TABLESPACE MAXDATA PAGESIZE 32 K MANAGED BY AUTOMATIC STORAGE INITIALSIZE {{ db2_tablespace_size }} BUFFERPOOL MAXBUFPOOL
```

Comment:
@caroazad instead of using the same variable to configure both tablespaces, let's split them in two as users can set different sizes depending on the tablespace created:

- `db2_tablespace_data_size`
- `db2_tablespace_index_size`

Reply:
Sure, I will do that asap

Reply:
Done